src/main/java/uk/gov/hmcts/ccd/domain/model/search/CaseSearchResult.java

```diff
@@ -1,25 +1,38 @@
 package uk.gov.hmcts.ccd.domain.model.search;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
+
+import java.util.ArrayList;
 import java.util.List;
 
 import static java.util.Collections.emptyList;
 import static java.util.stream.Collectors.toList;
 
-import uk.gov.hmcts.ccd.domain.model.definition.CaseDetails;
-
 public class CaseSearchResult {
-
     public static final CaseSearchResult EMPTY = new CaseSearchResult(0L, emptyList());
 
     private Long total;
     private List<CaseDetails> cases;
+    @JsonProperty("case_types_results")
+    private List<CaseTypeResults> caseTypeResults;
 
     public CaseSearchResult() {
     }
 
-    public CaseSearchResult(Long total, List<CaseDetails> cases) {
+    public CaseSearchResult(Long total, List<CaseDetails> cases, List<CaseTypeResults> caseTypeResults) {
         this.cases = cases;
         this.total = total;
+        this.caseTypeResults = caseTypeResults;
+    }
+
+    public CaseSearchResult(Long total, List<CaseDetails> cases) {
+        this(total, cases, new ArrayList<>());
+    }
+
+
+    public CaseSearchResult(List<CaseTypeResults> caseTypeResults, Long total, List<CaseDetails> cases) {
```

Comment:
Can we remove this constructor? It looks like the same as the one on L23 except with arguments in a different order?

Reply:
DONE.